docs/manual/ar/introduction/Creating-text.html

```diff
@@ -89,7 +89,7 @@ <h2>5. خطوط نقطية (Bitmap Fonts)</h2>
 		<div>
 			<p>
 				تسمح BMFonts (الخطوط النقطية) بدمج الصور الرمزية في BufferGeometry واحد. يدعم عرض BMFont التفاف الكلمات ، وتباعد الأحرف ، وتقنين الأحرف ، وحقول المسافة الموقعة مع المشتقات القياسية ، وحقول المسافة الموقعة متعددة القنوات ، والخطوط متعددة الأنسجة ، والمزيد.
-				انظر [link:https://github.com/Jam3/three-bmfont-text three-bmfont-text].
+				انظر [link:https://github.com/Jam3/three-bmfont-text three-bmfont-text] or [link:https://github.com/felixmariotto/three-mesh-ui three-mesh-ui].
```

Comment:
I think you need to change the order on this one?

Reply:
I know that Arabic is read from right to left, but I'm very confused by the different Editing and Viewing order with it.
If look at the overall right-to-left order, it is right now, so I deliberately used this order which different from `en` and `zh`.
But from the single piece of English `three-bmfont-text or three-mesh-ui`, the order is wrong.
![image](https://user-images.githubusercontent.com/10785634/104009274-bde20900-51e5-11eb-87ce-433ac296e9a8.png)
![image](https://user-images.githubusercontent.com/10785634/104009298-c63a4400-51e5-11eb-9342-0dc7f6ce1f35.png)

I don't know which one is correct, maybe all wrong.
So I'll changed to using Arabic word of `or`.
![image](https://user-images.githubusercontent.com/10785634/104009780-81fb7380-51e6-11eb-8c31-201a9888f7d6.png)
![image](https://user-images.githubusercontent.com/10785634/104009959-c25af180-51e6-11eb-97ad-872c0b912145.png)

![image](https://user-images.githubusercontent.com/10785634/104009923-b838f300-51e6-11eb-98be-09a576784dae.png)

Reply:
That's a rabbit hole I prefer not to venture into 😅